getting-started/ts-example/README.md

```diff
@@ -73,13 +73,13 @@ Create a file named `tracing.ts` and add the following code:
 
 ```typescript
 import * as opentelemetry from '@opentelemetry/core';
```

Comment:
I think this should point to `@opentelemetry/api` and we need `@opentelemetry/core` also to get `LogLevel`

Reply:
Fixed in all examples